pkgs/by-name/co/cosmic-edit/package.nix

```diff
@@ -0,0 +1,97 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  rust,
+  rustPlatform,
+  cmake,
+  makeBinaryWrapper,
+  cosmic-icons,
+  just,
+  pkg-config,
+  libxkbcommon,
+  glib,
+  gtk3,
+  libinput,
+  fontconfig,
+  freetype,
+  wayland,
+  xorg,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cosmic-edit";
+  version = "unstable-2023-11-29";
+
+  src = fetchFromGitHub {
+    owner = "pop-os";
+    repo = "pname";
```

Comment:
Noob question: should this be a string or the accessor to `pname`?

ie. repo = pname; or repo = "$pname"; or similar.

Reply:
Ah it should just be pname:

https://github.com/NixOS/nixpkgs/pull/265091/files

Good catch.

Reply:
Fixed now!